content/800-guides/240-partner-api.mdx

```diff
@@ -0,0 +1,193 @@
+---
+title: 'How to use the Prisma Postgres Partner API'
+metaTitle: 'How to use the Prisma Postgres Partner API'
+description: 'Learn how to use the Prisma Postgres Partner API'
+sidebar_label: 'Partner API'
+image: '/img/guides/prisma-partner-api-cover.png'
+community_section: true
+---
+
+## Introduction
+
+This guide walks you through how to use the Prisma Postgres Partner API, which powers the [`npx create-db`](https://create-db.prisma.io/) command.
+
+You'll learn how a Prisma Postgres database is provisioned, how you can transfer it to a specific workspace, and how the process is secured using OAuth2. By the end, you'll understand the full flow and how to leverage it in your own projects.
+
+This guide references the actual implementation in the `npx create-db` CLI and Cloudflare Workers for a real world example. The repo for the `npx create-db` is [here](https://github.com/prisma/create-db) which can be used as a reference for how to use the Partner API in your own projects.
+
+## Overview
+
+The `npx create-db` CLI command provisions a temporary Prisma Postgres database using the Prisma Postgres Partner API. The process is handled by two Cloudflare Workers, [create-db-worker](https://github.com/prisma/create-db/tree/main/create-db-worker) and [claim-db-worker](https://github.com/prisma/create-db/tree/main/claim-db-worker), with the [create-db CLI](https://github.com/prisma/create-db/tree/main/create-db) acting as the entry point:
+
+- **`create-db-worker`**: Provisions the database and schedules deletion.
+- **`claim-db-worker`**: Handles the claim and transfer process, including OAuth2 authentication.
+- **`create-db` CLI**: The entry point for the CLI command.
+
+## 1. Creating a database
```

Comment:
We'll also need a step before this that explains how to set yourself up as a partner, such as creating an account, getting OAuth client ID and secret from us (we can update this later with the self-serve flow), etc.

Should also link somewhere to the full management API docs so they can see all the endpoints there.